src/core/packages/ui-settings/browser-internal/src/ui_settings_client.test.ts

```diff
@@ -240,8 +240,8 @@ describe('#isDefault', () => {
 
   it('returns false if name is not known', () => {
     const { client } = setup();
-    client.set('dateFormat', 'foo');
-    expect(client.isDefault('dateFormat')).toBe(false);
+    client.set('unknownSettingKey', 'foo');
+    expect(client.isDefault('unknownSettingKey')).toBe(false);
```

Comment:
The test case was wrong, the only reason it was passing was because a value was being set so it's value was not the default anymore